self-core-impl/src/main/java/com/selfxdsd/core/BaseSelf.java

```diff
@@ -53,4 +55,10 @@
     Storage storage() {
         return this.storage;
     }
+
+    @Override
+    public User login(final Login login) {
+        return login.signUp(storage());
```

Comment:
@criske poti pune aici ``this.storage()``, te rog? E mai clar unde se afla metoda asta, ``storage()``.